findphotodates.py

```
@@ -647,34 +647,53 @@ def run_scan(directory, output, extensions, locate=False, quiet=False, debug=Fal
                     files_to_process.append((out_path, key_path, size_bytes, mtime_ns))
             except OSError as e:
                 error_log.write(f"Warning: Could not stat '{file}': {str(e)}\n")
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)